model/shared/src/main/scala/explore/model/ObsSummary.scala

```diff
@@ -131,7 +131,17 @@ object ObsSummaryWithTitleConstraintsAndConf {
   val activeStatus = Focus[ObsSummaryWithTitleConstraintsAndConf](_.activeStatus)
 
   implicit val eqObsSummaryWithTitleConstraintsAndConf: Eq[ObsSummaryWithTitleConstraintsAndConf] =
-    Eq.by(o => (o.id, o.title, o.subtitle, o.constraints, o.status, o.activeStatus, o.duration))
+    Eq.by(o =>
+      (o.id,
+       o.title,
+       o.subtitle,
+       o.constraints,
+       o.status,
+       o.activeStatus,
+       o.duration,
+       o.scienceMode
```

Comment:
Do you know if this `scienceMode` is used elswewhere? Or is it only for the purpose of sending to the AladinContainer?

Reply:
It is (or will be used for step generation)

Reply:
The value in the ObsSummary will be used? It's also in the ObsEditQuery.